src/interfaces/external/Morpho/IMorpho.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+interface IMorpho {
```

Comment:
These don't seem to line up completely with the function sigs shown in morpho repo, but I could be mistaken. Please clarify. Why not have two separate Morpho interfaces? (AAVE v2 - blob/main/src/aave-v2/interfaces/IMorpho.sol & AAVE v3 - https://github.com/morpho-org/morpho-aave-v3/blob/main/src/interfaces/IMorpho.sol)